tests/src/Simple/SimpleTest.targets

```diff
@@ -59,4 +59,8 @@
     <IlcArg Include="--stacktracedata" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ReadyToRunMultiModuleAssemblies Include="$(MSBuildThisFileDirectory)ReadyToRunMultiModule\bin\$(Configuration)\$(Platform)\MultiModuleTestingClass.dll;$(MSBuildThisFileDirectory)ReadyToRunMultiModule\bin\$(Configuration)\$(Platform)\ReadyToRunMultiModule.exe" />
```

Comment:
Why is this in `SimpleTest.targets` (included by all tests) instead of in the project file(s) that need this? This looks fragile.